### DISCUSSION TITLE
NextAuth with NextJS 13.2.1 Route Handlers

### DISCUSSION THREAD

**Discussion by fachrur95** [nextauthjs/next-auth#6818] [Help]
I have issue about nextAuth nextjs 13.2.1 about "/api/auth/session" Method Not Allowed. Change moved from pages/api/auth/[...nextauth].ts to /app/api/auth/[...nextauth]/route.ts. Any solution to solve this? maybe I wrong get the new concept. Thank you everyone

**Comment by bessim-dev** [comment]
same question

**Comment by benderillo** [comment]
The next-auth does not yet run in the Route Handlers. 
This is work in progress.
You can see how it meant to be migrated here: https://github.com/nextauthjs/next-auth/blob/93d22520137af948c494d606b9173b5bba4e181b/apps/dev/app/api/auth/%5B...nextauth%5D/route.ts

The PR for this support is here: https://github.com/nextauthjs/next-auth/pull/6777 
The ticket against next-auth: https://github.com/nextauthjs/next-auth/issues/6792
At the moment the PR is blocked by an issue in NextJS that does not allow to set multiple cookies on a response
